libs/common/src/lib/generate-id.ts

```diff
@@ -1,5 +1,16 @@
-// Simple alphanumeric ID: https://stackoverflow.com/a/12502559/453959
-// This isn't a real GUID!
-export function generateId(): string {
-    return Math.random().toString(36).slice(2);
+/*
+Implementation was copied from Nano ID (https://github.com/ai/nanoid)
+For more details, see LICENSE
+ */
+export const urlAlphabet = 'useandom-26T198340PX75pxJACKVERYMINDBUSHWOLF_GQZbfghjklqvwyzrict';
+
+export function generateId(size = 6): string {
```

Comment:
We can hardcode the size to 6 inside the function. No parameter, no size fallback needed.

More lightweight, more mini 😀

Reply:
Done :)